plugin.cpp

```diff
@@ -27,72 +28,77 @@ static const char * def_cfg = QUOTE({
 	"email_to" : {
 		"description" : "The address to send the alert to",
 		"type" : "string",
-		"default" : "alert.subscriber@dianomic.com",
+		"default" : "Notification alert subscriber <alert.subscriber@dianomic.com>",
 		"order" : "1",
-		"displayName" : "To address"
+		"displayName" : "To"
 		},
-	"email_to_name" : {
-		"description" : "The name to send the alert to",
+	"email_cc" : {
+		"description" : "The address to send the alert to Cc",
 		"type" : "string",
-		"default" : "Notification alert subscriber",
+		"default" : "Notification alert subscriber <alert.subscriber@dianomic.com>",
 		"order" : "2",
-		"displayName" : "To "
+		"displayName" : "CC"
 		},
-	"subject" : {
-		"description" : "The email subject",
+	"email_bcc" : {
+		"description" : "The address to send the alert to Bcc",
 		"type" : "string",
-		"displayName" : "Subject",
+		"default" : "Notification alert subscriber <alert.subscriber@dianomic.com>",
 		"order" : "3",
-		"default" : "Fledge alert notification"
+		"displayName" : "BCC"
 		},
 	"email_from" : {
 		"description" : "The address the email will come from",
 		"type" : "string",
 		"displayName" : "From address",
-		"default" : "dianomic.alerts@gmail.com",
+		"default" : "Notification Alert <dianomic.alerts@gmail.com>",
 		"order" : "4"
 		},
-	"email_from_name" : {
-		"description" : "The name used to send the alert email",
+	"subject" : {
+		"description" : "The email subject. Macro $NOTIFICATION_INSTANCE_NAME$ can be used to provide information about notification instance name",
 		"type" : "string",
-		"default" : "Notification alert", 
-		"displayName" : "From name",
-		"order" : "5"
+		"displayName" : "Subject",
+		"order" : "5",
+		"default" : "Fledge alert notification triggerd by $NOTIFICATION_INSTANCE_NAME$ "
```

Comment:
Don't use the word triggered as if it is sent by a toggle notification it can be sent when the condition is triggered or cleared.